grammar/exp.js

```diff
@@ -282,11 +227,33 @@ module.exports = {
    * This is left-associative, although in reality this would depend on the fixity declaration for the operator.
    * The default is left, even though the reference specifies it the other way around.
    * In any case, this seems to be more stable.
+   *
+   * Also see `exp_section_left` comment.
    */
-  exp_infix: $ => seq($._exp_infix, $._qop, $._lexp),
+  exp_infix: $ =>
+    prec(1, seq(
+      $._exp_infix,
+      $._q_op,
+      $._lexp
+    )),
```

Comment:
```suggestion
  exp_infix: $ =>
    prec(1, seq(
      $._lexp,
      $._q_op,
      $._exp_infix
    )),
```
Letting this be right-associative seems to fix my issues. I don't know if it breaks anything else.

Reply:
Hmm I'm getting no changes after a fresh clone of the branch at cc14b26. The tests still pass though.